src/SeqViz/Linear/SeqBlock/Translations.tsx

```diff
@@ -3,44 +3,63 @@ import * as React from "react";
 import { borderColorByIndex, colorByIndex } from "../../../utils/colors";
 import randomid from "../../../utils/randomid";
 
-export default class TranslationRows extends React.PureComponent {
+export interface Translation {
+  id: string;
+  start: number;
+  end: number;
+  AAseq: string;
+  direction: -1 | 1;
+}
+
+export type inputRefFuncType = <T>(id: string, ref: unknown) => React.LegacyRef<T>;
+
+export type findXandWidthType = (
+  n1: number | undefined,
+  n2?: number
+) => {
+  x: number;
+  width: number;
+};
```

Comment:
These should be exported from where that method is defined in `SeqBlock`. Or, if there's a dependency cycle there, from a types-only file in the `SeqBlock` dir